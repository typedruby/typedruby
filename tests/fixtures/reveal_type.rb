def reveal => nil
  x = [1, "hello"]
  reveal_type(x)
  reveal_type
  nil
end
