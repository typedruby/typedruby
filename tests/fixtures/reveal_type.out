error: Revealed type is: [Integer, String]

        @ __ROOT__/tests/fixtures/reveal_type.rb:3
      3 |    reveal_type(x)
                         ^ expression

error: Too few arguments supplied

        @ __ROOT__/tests/fixtures/reveal_type.rb:4
      4 |    reveal_type
             ^^^^^^^^^^^ in this invocation
        @ __ROOT__/tests/fixtures/reveal_type.rb:4
      4 |    reveal_type
             ^^^^^^^^^^^ for this prototype
